winpr/libwinpr/utils/asn1/asn1.c

```diff
@@ -686,9 +686,9 @@ size_t WinPrAsn1EncOID(WinPrAsn1Encoder* enc, const WinPrAsn1_OID* oid)
 	return WinPrAsn1EncMemoryChunk(enc, ER_TAG_OBJECT_IDENTIFIER, oid);
 }
 
-size_t WinPrAsn1EncOctetString(WinPrAsn1Encoder* enc, const WinPrAsn1_OctetString* octets)
+size_t WinPrAsn1EncOctetString(WinPrAsn1Encoder* enc, const WinPrAsn1_OctetString* oid)
```

Comment:
This renaming is unfortunate, octets strings != oid

Reply:
ok, then we should rename the other way around.